sdcm/logcollector.py

```diff
@@ -978,7 +1012,7 @@ class JepsenLogCollector(LogCollector):
     cluster_log_type = "jepsen-data"
     cluster_dir_prefix = "jepsen-data"
 
-    def collect_logs(self, local_search_path: Optional[str] = None) -> Optional[str]:
+    def collect_logs(self, local_search_path: Optional[str] = None) -> Optional[list[str]]:
         s3_link = None
```

Comment:
s3_link = []
....
....
return s3_link

Reply:
Thanks. Fixed

Reply:
don't see it